examples/custom-module/README.md

```diff
@@ -26,3 +26,39 @@ And see in the apicast output:
 ```
 2016/11/16 16:52:00 [warn] 98009#0: *5 [lua] verbose.lua:7: call(): upstream response time: 0.001 upstream connect time: 0.000 while logging request, client: 127.0.0.1, server: _, request: "GET /?user_key=foo HTTP/1.1", upstream: "http://127.0.0.1:8081/?user_key=foo", host: "echo"
 ```
+
+## Writing own module
+
+There is example module of IP blacklist in [`blacklist.lua`](blacklist.lua).
+
+To honour the module inheritance, but still be able to override some methods from the `apicast` module, you'll
+need some clever use of metatables. Here a recommended skeleton of the module inheritance:
+
+```lua
+-- load and initialize the parent module
+local apicast = require('apicast').new()
+
+-- this is used in User-Agent
```

Comment:
Well the used module _NAME field is going to get used in User-Agent header from APIcast connecting to backend.